test/fake-timers-test.js

```diff
@@ -3221,7 +3221,7 @@ describe("FakeTimers", function () {
             assert.equals(fakeDateStr, new this.clock.Date().toString());
         });
 
-        it("mirrors native Date.prototype", function () {
+        it.skip("mirrors native Date.prototype", function () {
```

Comment:
I guess we could change this to be a test that it passes an instanceof check. `it("is a Date instance", ...`?

Reply:
Just kill the test. It would just be this in the end:

```
node -p 'class FooDate extends Date {}; D=Date; global.Date = function(){};  (new FooDate()) instanceof D'
true
```